src/main.ts

```diff
@@ -40,14 +40,46 @@ app.config.globalProperties.$filters = {
   formatDate(value: any, inFormat?: string, outFormat?: string) {
     // TODO Use Loxon instead
     // TODO Make default format configurable and from environment variables
-    return moment(value, inFormat).format(outFormat ? outFormat : 'MM-DD-YYYY');
+    return moment(value, inFormat).format(outFormat ? outFormat : 'hh:mm a');
```

Comment:
Why we have done this?
I think we should have a separate method for it

Reply:
Done